hyper_click.sublime-settings

```diff
@@ -55,17 +61,23 @@
     "less": ["less"],
     "php": ["php"],
     "stylus": ["styl", "stylus"],
-    "html": ["html"]
+    "html": ["html"],
+    "jstl": ["jsp", "tag"]
   },
   "default_filenames": {
     "js": ["index"]
   },
   "vendor_dirs": {
     "js": ["node_modules"],
-    "html": ["node_modules", "bower_components"]
+    "html": ["node_modules", "bower_components"],
+    "jstl": ["WEB-INF"]
   },
   "lookup_paths": {
-    "js": []
+    "js": [],
+    "jstl": [
+      "taglib *prefix ?= ?[\"'](.*)[\"'] *tagdir ?= ?[\"'](.*)[\"']",
+      "xmlns:(\\w*)=[\"']urn:jsptagdir:(.*?)[\"']"
+    ]
```

Comment:
looks like these regex patterns are not paths, but something domain-specific. 
I suggest add dedicated domain-specific property for that, for example

```json
"jstl_dirs_regex": []
```

or

```json
"jstl": {
  "dirs_regex": []
}
```

Reply:
Hi, the regex is a costructed path
*<%@ taglib prefix="common" tagdir="/WEB-INF/tags/desktop/common" %>*

where *tagdir* is the target path based on the project **C:/.../.../.../WEB-INF/tags/desktop/common**
and prefix is the reference in the html file
*<common:content idtemplate="share-email-dynamic" htmltag="div"/>*

where **common** is the reference and **content** is the name of the file inside the above directory

while the *vendor_dirs* is the base root folder where you must start looking for files (generally match with the first part of the *tagdir*): 

so the file will be **C:/.../.../.../WEB-INF/tags/desktop/common/content.tag**

i will do a new field if necessary but as *lookup_paths* i understood that are the main dirs or folder where the file are positioned

Reply:
That makes sense. But probably later you will want add static lookup paths, and currently  there is no way to do it )

Up to you.